website/routes.py

```diff
@@ -197,6 +198,13 @@ def make_url_map(app):
             OsfWebRenderer('public/pages/meeting_landing.mako'),
         ),
 
+        Rule(
+            '/api/v1/citation_styles/',
```

Comment:
This can be moved to a call to `process_rules` that passes the `/api/v1` prefix. Why not put it next to the other citations route?